File: BE/health-service/src/main/java/com/s206/health/nutrition/meal/dto/response/MealDailyResponse.java
package com.s206.health.nutrition.meal.dto.response;

import lombok.Builder;
import lombok.Getter;


import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.List;

@Getter
@Builder
public class MealDailyResponse {
    private Integer mealId;
    private LocalDate mealDate;
    private Boolean isMeal;
    private MealNutritionDto totalNutrition;
    private List<MealTimeNutritionResponse> mealTimes;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: FE/mobile/src/main/java/com/example/diaviseo/network/meal/dto/res/MealDailyResponse.kt
package com.example.diaviseo.network.meal.dto.res

// 날짜별 식단 조회할 때 젤 처음 데이터 형식
data class MealDailyResponse(
    val mealId: Int?,
    val mealDate: String,
    val isMeal: Boolean?,
    val totalNutrition: MealNutritionResponse,
    val mealTimes: List<MealTimeNutritionResponse>,
    val createdAt: String,
    val updatedAt: String
)

